### ISSUE TITLE
My accuracy on dev just have 0.73?

### ISSUE THREAD

**Issue by yuye2133** [dhwajraj/deep-siamese-text-similarity#7]
I runned "python train.py", but the accuracy on dev was 0.73, and the loss was 0.08, it could not decrease. 
I runned it long times. but the accuracy and loss cannot have a large change.

**Comment by zhaozhichao4515** [comment]
I met same problem,have you solve it?

**Comment by jnAILab** [comment]
same problem...

**Comment by dminh** [comment]
Me too ... please help us!

**Comment by xiaoyugan0418** [comment]
I think there is something wrong in the train.py. The dropout_keep_prob should be set as 1, not FLAGS.dropout_keep_prob.

**Comment by willduan** [comment]
same problem

**Comment by willduan** [comment]
@xiaoyugan0418 I take your point.

**Comment by smartlan** [comment]
oh,seems "correct_predictions = tf.equal(self.distance, self.input_y)" this part  made a mistake

**Comment by willduan** [comment]
@smartlan I wonder what's the correct equation about 
`correct_predictions = tf.equal(self.distance, self.input_y)`?

**Comment by willduan** [comment]
@smartlan  I wonder what's the correct equation about
`correct_predictions = tf.equal(self.distance, self.input_y)`?

**Comment by dhwajraj** [comment]
please try with fresh checkout for latest updates compatible with tensorflow v1.2.1

**Comment by Huijun-Cui** [comment]
why such a bad toy code attract so many people , i dont understand ,so many mistakes
